prime-router/settings/organizations.yml

```diff
@@ -97,12 +107,22 @@
       schemaName: direct/careevolution-covid-19
       format: HL7
 
-- name: "john_hopkins"
-  description: "John Hopkin's / IHS school testing program"
+- name: primary
```

Comment:
Primary is such a generic word.  Maybe primaryDiag?  Not sure what naming convention has been used.

Reply:
Good point.   my informal convention has been to use the shorthand name that companies use to refer to themselves